froide/helper/form_utils.py

```diff
@@ -18,9 +19,35 @@ def get_data(error):
     return error.get_json_data()
 
 
+# FIXME WIP this is maybe not the smartest way to do this
+# use simplejson instead?
+def serialize_extra(obj):
+    if isinstance(obj, Decimal):
+        return {
+            "__Decimal": True,
```

Comment:
is this a nice way or should we add a tag field that specifies the type of the object serialized here? e.g. something like

```suggestion
            "__type__": "__Decimal",
```

Reply:
i like your suggestion; tbh i don't use the type info anywhere - i just felt dirty not doing anything :) yagni?